tests/categorical/test_contingency.py

```diff
@@ -227,6 +229,12 @@ def test_dimension_broadcasting():
     assert accuracy.dims == ("time",)
     assert len(accuracy.time) == 5
 
+    assert accuracy[0] == (9 + 6) / 18
+    assert accuracy[1] == (10 + 4) / 18
+    assert accuracy[2] == (10 + 1) / 18
+    assert accuracy[3] == (10 + 0) / 18
+    assert accuracy[4] == (10 + 0) / 18
```

Comment:
Consider creating an xarray object with this data in it and replace the asserts from line 229-236 with a single `xr.testing.assert_equal`. This would also test that the coordinates are correct.